deploy/dev/5_load_environment.sh

```diff
@@ -8,9 +8,20 @@ function main() {
 
   set_namespace "$APP_NAMESPACE_NAME"
 
-  configure_secret
+  if [ "${DEV}" = "false" ]; then
+    configure_secret
```

Comment:
you have `configure_secret` and `create_k8s_secret`. please align them (i would choose `configure_k8s_secret`)

Reply:
I don't want to align them because their purposes are very different.
`configure_secret` is for non-helm tests. It sets a different path and most importantly, `create_k8s_resource` creates roles and rolebindings with `sh.yml` files

Which is a very different flow than what we want for Helm. For these reasons I want them separate. I can definitely update the naming though